tests/positive/general/preemption.go

```diff
@@ -23,29 +23,21 @@ import (
 )
 
 func init() {
-	// Default config is applied
-	register.Register(register.PositiveTest, makePreemptTest("D"))
-	// Base and default configs are both applied
-	register.Register(register.PositiveTest, makePreemptTest("BD"))
 	// Base and provider configs are both applied
 	register.Register(register.PositiveTest, makePreemptTest("BP"))
-	// Provider config is applied; default config is ignored
-	register.Register(register.PositiveTest, makePreemptTest("dP"))
```

Comment:
Thinking we can drop the tests that included a default config - the platform config `p` is the only one that can be ignored (by passing a user config `U`), which has a test case already.